templates/slurm.conf.j2

```diff
@@ -103,31 +103,31 @@ PropagateResourceLimitsExcept=MEMLOCK
 AccountingStorageType=accounting_storage/filetxt
 Epilog=/etc/slurm/slurm.epilog.clean
 {% for part in openhpc_slurm_partitions %}
-{% for group in part.get('groups', [part]) %}
-{% set group_name = group.cluster_name|default(openhpc_cluster_name) ~ '_' ~ group.name %}
-{# If using --limit, the first host in each group may not have facts available. Find one that does. #}
-{% set group_hosts = groups[group_name] | intersect(play_hosts) %}
-
-NodeName=DEFAULT State=UNKNOWN RealMemory={% if 'ram_mb' in group %}{{group.ram_mb}}{% else %}1{% endif %}{% if group_hosts | length > 0 %}\
-{% set first_host_hv = hostvars[group_hosts | first] %}
+{%   for group in part.get('groups', [part]) %}
+{%      set group_name = group.cluster_name|default(openhpc_cluster_name) ~ '_' ~ group.name %}
+{#      If using --limit, the first host in each group may not have facts available. Find one that does, or die: #}
+{%      set group_hosts = groups[group_name] | intersect(play_hosts) %}
```

Comment:
Wondered if we should allow for a static override of this part of the config so that you don't need a compute host in the limit. Probably for another PR though...

Reply:
Apparently there is some sneaky way (used in Kayobe?) of getting facts for hosts not in the play... but TBH I'm not sure that's necessary here so I thought I'd start simple.